[2024-05-06T11:39:32.443+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-06T11:39:32.469+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_operator_dag.create_pet_table_mysql manual__2024-05-06T11:29:02.907851+00:00 [queued]>
[2024-05-06T11:39:32.480+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_operator_dag.create_pet_table_mysql manual__2024-05-06T11:29:02.907851+00:00 [queued]>
[2024-05-06T11:39:32.481+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 3
[2024-05-06T11:39:32.499+0000] {taskinstance.py:2327} INFO - Executing <Task(MySqlOperator): create_pet_table_mysql> on 2024-05-06 11:29:02.907851+00:00
[2024-05-06T11:39:32.506+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1627) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-06T11:39:32.508+0000] {standard_task_runner.py:63} INFO - Started process 1628 to run task
[2024-05-06T11:39:32.508+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag', 'create_pet_table_mysql', 'manual__2024-05-06T11:29:02.907851+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/kachevo.py', '--cfg-path', '/tmp/tmpjh7p68mk']
[2024-05-06T11:39:32.511+0000] {standard_task_runner.py:91} INFO - Job 150: Subtask create_pet_table_mysql
[2024-05-06T11:39:32.559+0000] {task_command.py:426} INFO - Running <TaskInstance: postgres_operator_dag.create_pet_table_mysql manual__2024-05-06T11:29:02.907851+00:00 [running]> on host db2363de7199
[2024-05-06T11:39:32.641+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='postgres_operator_dag' AIRFLOW_CTX_TASK_ID='create_pet_table_mysql' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T11:29:02.907851+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T11:29:02.907851+00:00'
[2024-05-06T11:39:32.643+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-06T11:39:32.653+0000] {sql.py:276} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS pet (
        pet_id SERIAL PRIMARY KEY,
        name VARCHAR NOT NULL,
        pet_type VARCHAR NOT NULL,
        birth_date DATE NOT NULL,
        OWNER VARCHAR NOT NULL);
        
[2024-05-06T11:39:32.661+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2024-05-06T11:39:32.669+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2024-05-06T11:39:32.685+0000] {sql.py:457} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS pet (
        pet_id SERIAL PRIMARY KEY,
        name VARCHAR NOT NULL,
        pet_type VARCHAR NOT NULL,
        birth_date DATE NOT NULL,
        OWNER VARCHAR NOT NULL);
        , parameters: None
[2024-05-06T11:39:32.686+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-06T11:39:32.687+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 282, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 404, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 462, in _run_command
    cur.execute(sql_statement)
  File "/home/airflow/.local/lib/python3.12/site-packages/MySQLdb/cursors.py", line 179, in execute
    res = self._query(mogrified_query)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/MySQLdb/cursors.py", line 330, in _query
    db.query(q)
  File "/home/airflow/.local/lib/python3.12/site-packages/MySQLdb/connections.py", line 261, in query
    _mysql.connection.query(self, query)
MySQLdb.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'NOT NULL,\n        pet_type VARCHAR NOT NULL,\n        birth_date DATE NOT NULL,\n ' at line 3")
[2024-05-06T11:39:32.696+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=postgres_operator_dag, task_id=create_pet_table_mysql, execution_date=20240506T112902, start_date=20240506T113932, end_date=20240506T113932
[2024-05-06T11:39:32.708+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 150 for task create_pet_table_mysql ((1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'NOT NULL,\n        pet_type VARCHAR NOT NULL,\n        birth_date DATE NOT NULL,\n ' at line 3"); 1628)
[2024-05-06T11:39:32.722+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-06T11:39:32.731+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1296: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-05-06T11:39:32.732+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1296: AirflowProviderDeprecationWarning: Call to deprecated class MySqlOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-05-06T11:39:32.745+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-06T11:39:32.748+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
